en/docs/catalogs/crd.md

```diff
@@ -0,0 +1,4359 @@
+                </table>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>status</code></br>
+                <em>
+                    <a href="#dp.wso2.com/v1alpha1.TokenIssuerStatus">
+                        TokenIssuerStatus
+                    </a>
+                </em>
+            </td>
+            <td>
+            </td>
+        </tr>
+    </tbody>
+</table>
+<h3 id="dp.wso2.com/v1alpha1.APIAuth">APIAuth
+    <a class="headerlink" href="#dp.wso2.com%2fv1alpha1.APIAuth" title="Permanent link">¶</a>
+</h3>
+<p>
+    (<em>Appears on:</em>
+    <a href="#dp.wso2.com/v1alpha1.AuthSpec">AuthSpec</a>)
+</p>
+<p>
+<p>APIAuth Authentication scheme type and details</p>
+</p>
+<table>
+    <thead>
+        <tr>
+            <th>Field</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <code>oauth2</code></br>
+                <em>
+                    <a href="#dp.wso2.com/v1alpha1.Oauth2Auth">
+                        Oauth2Auth
+                    </a>
+                </em>
+            </td>
+            <td>
+                <em>(Optional)</em>
+                <p>Oauth2 is to specify the Oauth2 authentication scheme details</p>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>apiKey</code></br>
+                <em>
+                    <a href="#dp.wso2.com/v1alpha1.APIKeyAuth">
+                        []APIKeyAuth
+                    </a>
+                </em>
+            </td>
+            <td>
+                <em>(Optional)</em>
+                <p>APIKey is to specify the APIKey authentication scheme details</p>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>testConsoleKey</code></br>
+                <em>
+                    <a href="#dp.wso2.com/v1alpha1.TestConsoleKeyAuth">
+                        TestConsoleKeyAuth
+                    </a>
+                </em>
+            </td>
+            <td>
+                <em>(Optional)</em>
+                <p>TestConsoleKey is to specify the Test Console Key authentication scheme details</p>
+            </td>
+        </tr>
+    </tbody>
+</table>
+<h3 id="dp.wso2.com/v1alpha1.APIKeyAuth">APIKeyAuth
+    <a class="headerlink" href="#dp.wso2.com%2fv1alpha1.APIKeyAuth" title="Permanent link">¶</a>
+</h3>
+<p>
+    (<em>Appears on:</em>
+    <a href="#dp.wso2.com/v1alpha1.APIAuth">APIAuth</a>)
+</p>
+<p>
+<p>APIKeyAuth APIKey Authentication scheme details</p>
+</p>
+<table>
+    <thead>
+        <tr>
+            <th>Field</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <code>in</code></br>
+                <em>
+                    string
+                </em>
+            </td>
+            <td>
+                <pre><code>In is to specify how the APIKey is passed to the request
+</code></pre>
```

Comment:
Shouldn't the in field be just a `<p>` tag like the name field below that?

<img width="855" alt="image" src="https://github.com/wso2/docs-apk/assets/30475839/112136ad-7519-47e0-a42f-618c20a9ef28">

Reply:
Fixed in PR https://github.com/wso2/docs-apk/pull/584